app/agent/rest/host.js

```diff
@@ -3,12 +3,12 @@ angular.module('portainer.agent').factory('Host', [
   function AgentFactory($resource, API_ENDPOINT_ENDPOINTS, EndpointProvider) {
     'use strict';
     return $resource(
-      API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/host/:action',
+      API_ENDPOINT_ENDPOINTS + '/:endpointId/docker/v:version/host/:action',
       {
         endpointId: EndpointProvider.endpointID
```

Comment:
What about passing the version parameter as a global parameter instead of a per method parameter? (like it's done for browse)